src/lightning_lite/strategies/ddp.py

```diff
@@ -53,6 +61,7 @@ def __init__(
         precision: Optional[Precision] = None,
         process_group_backend: Optional[str] = None,
         timeout: Optional[timedelta] = default_pg_timeout,
+        start_method: Literal["popen", "spawn", "fork", "forkserver"] = "popen",
```

Comment:
The three options from the spawn strategy and the popen from the normal ddp strategy are now all available to choose from here. The previous `DDPSpawnStrategy` becomes `DDPStrategy(start_method="spawn")`